### PR TITLE
Fix references to bundle and minor fmt fix for kpt

### PR DESCRIPTION
**Pull request by ajayk**
Fix references to bundle and minor fmt fix for kpt
Signed-off-by: ajayk <ajaykemparaj@gmail.com>

### CODE REVIEW

File: bundles/cis-k8s-v1.5.1/Kptfile [hunk 1]
```diff
@@ -6,6 +6,6 @@ info:
   description: CIS Benchmark Kubernetes v1.5.1
 pipeline:
   mutators:
-  - image: gcr.io/kpt-fn/set-enforcement-action:v0.1
-    configMap:
-      enforcementAction: dryrun
+    - image: gcr.io/kpt-fn/set-enforcement-action:v0.1
```
Comment (g-awmalik) [inline bundles/cis-k8s-v1.5.1/Kptfile:9]: Why do we need to indent this? It works fine the way it is and a valid yaml format.

**Review by g-awmalik — CHANGES_REQUESTED** [review]
Thanks for catching the typo in README.md. Please review the comments.

File: bundles/cis-k8s-v1.5.1/Kptfile (hunk 1 above)
Comment (ajayk) [reply bundles/cis-k8s-v1.5.1/Kptfile:9]: when i init the kpt , kpt is mutating the file with the indentations , you can reproduce by just initing 

```

➜  bundles git:(master) kpt live init cis-k8s-v1.5.1
initializing Kptfile inventory info (namespace: default)...success
➜  bundles git:(master) ✗ git diff


diff --git a/bundles/cis-k8s-v1.5.1/Kptfile b/bundles/cis-k8s-v1.5.1/Kptfile
index 741b1c3..3ed9f1b 100644
--- a/bundles/cis-k8s-v1.5.1/Kptfile
+++ b/bundles/cis-k8s-v1.5.1/Kptfile
@@ -6,6 +6,10 @@ info:
   description: CIS Benchmark Kubernetes v1.5.1
 pipeline:
   mutators:
-  - image: gcr.io/kpt-fn/set-enforcement-action:v0.1
-    configMap:
-      enforcementAction: dryrun
+    - image: gcr.io/kpt-fn/set-enforcement-action:v0.1
+      configMap:
+        enforcementAction: dryrun
+inventory:
+  namespace: default
+  name: inventory-10646875
+  inventoryID: 86b9dd86e4b3fafc693c789e563545bb8973942c-1648227623537222000
````

Comment (g-awmalik) [reply bundles/cis-k8s-v1.5.1/Kptfile:9]: I see the problem now.

**Review by g-awmalik — APPROVED** [review]

**Review by g-awmalik — APPROVED** [review]

**Pull request merged**